tests/t0124_gateway_ipns_record_test.go

```diff
@@ -0,0 +1,115 @@
+package tests
+
+import (
+	"testing"
+
+	"github.com/ipfs/gateway-conformance/tooling/car"
+	. "github.com/ipfs/gateway-conformance/tooling/check"
+	"github.com/ipfs/gateway-conformance/tooling/ipns"
+	"github.com/ipfs/gateway-conformance/tooling/test"
+	. "github.com/ipfs/gateway-conformance/tooling/test"
+)
+
+func TestGatewayIPNSRecord(t *testing.T) {
+	fixture := car.MustOpenUnixfsCar("t0124/fixtures.car")
+	file := fixture.MustGetRoot()
+	fileCID := file.Cid()
+
+	// test_expect_success "Add the test directory & IPNS records" '
+	// ipfs dag import ../t0124-gateway-ipns-record/fixtures.car &&
+	// ipfs routing put /ipns/${IPNS_KEY} ../t0124-gateway-ipns-record/${IPNS_KEY}.ipns-record
+	// '
+	// TODO: what if you'd import with `t0124/simple-id.ipns-record`,
+	// then the loader "just" finds the file with `peer_id_simple-id.ipns-record`?
+	ipns := ipns.MustOpenIPNSRecordWithKey("t0124/k51qzi5uqu5dh71qgwangrt6r0nd4094i88nsady6qgd1dhjcyfsaqmpp143ab.ipns-record")
```

Comment:
Is this `TODO` still relevant? Or is "simple ID" something else?

Reply:
Thank for the check, it's outdated, I learned later that we have to keep the pubkey around since it's not always in the ipns-record itself.